source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluatorEmployeesCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using CommonJobs.Application.EvalForm.EmployeeSearching;
+using CommonJobs.Infrastructure.RavenDb;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm.Commands
+{
+    public class GetEvaluatorEmployeesCommand : Command<List<EmployeeEvaluationDTO>>
+    {
+        private string _responsible { get; set; }
+
+        public GetEvaluatorEmployeesCommand(string responsible)
+        {
+            _responsible = responsible;
+        }
+
+        public override List<EmployeeEvaluationDTO> ExecuteWithResult()
+        {
+            RavenQueryStatistics stats;
+            IQueryable<EmployeeToEvaluate_Search.Projection> query = RavenSession
+                .Query<EmployeeToEvaluate_Search.Projection, EmployeeToEvaluate_Search>()
+                .Statistics(out stats)
+                //.Where(e => e.Evaluators.Any(ev => ev == _responsible))
```

Comment:
I should've deleted the comment. The next line is what matters: .Where(e => e.Responsible == _responsible)